groups/live/profiles/heritage-live-eu-west-2/vars

```diff
@@ -0,0 +1,50 @@
+# Account details
+aws_profile = "heritage-live-eu-west-2"
+aws_region  = "eu-west-2"
+aws_account = "heritage-live"
+
+# Account shorthand
+account = "hlive"
+region  = "euw2"
+
+# Application details
+environment = "live"
+
+# Jfil Server
+jfil_application       = "Jfil Server"
+jfil_ec2_name          = "jfil-app"
+jfil_ec2_instance_type = "t2.xlarge"
+jfil_ami               = "ami-027676865ead3918d" # rhel6-base-0.1.25
+
+jfil_cw_logs = {
```

Comment:
Live env, will need longer than 30 days log retention?

Reply:
not sure on this but happy to take steer on longer retention.  I took this from other services in live (ewf)

Reply:
spoken to Matt on Slack.  He's happy that I revisit this, so long as prior to go-live